pkg/storaged/stratis-details.jsx

```diff
@@ -181,9 +190,27 @@ export const StratisPoolDetails = ({ client, pool }) => {
                       pool.ClevisInfo[0] && // pool has consistent clevis config
                       (!pool.ClevisInfo[1][0] || pool.ClevisInfo[1][1][0] == "tang")); // not bound or bound to "tang"
     const tang_url = can_tang && pool.ClevisInfo[1][0] ? JSON.parse(pool.ClevisInfo[1][1][1]).url : null;
+    const blockdevs = client.stratis_pool_blockdevs[pool.path] || [];
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19079-20230809-122740-f1602c20-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)